src/test/java/com/kobylynskyi/graphql/codegen/kotlin/GraphQLCodegenGitHubTest.java

```diff
@@ -188,6 +189,26 @@ void generate_CustomFieldsResolvers() throws Exception {
                 getFileByName(files, "AcceptTopicSuggestionPayloadResolver.kt"));
     }
 
+    @Test
+    void generate_CustomFieldsResolversWithAnnotations() throws Exception {
+        mappingConfig.setModelNamePrefix("Github");
+        mappingConfig.setModelNameSuffix("TO");
+        mappingConfig.setApiNameSuffix("WithAnnotation");
+        mappingConfig.setResolverArgumentAnnotations(singleton("some.Annotation"));
```

Comment:
Can you add another test with multiple annotations? Thank you!

Reply:
Done (see commit e650281 below)